example/test.py

```diff
@@ -145,6 +168,43 @@ def test_expected_processes(self, web_container):
              '--bind unix:/var/run/gunicorn/gunicorn.sock --umask 0117'],
         ])))
 
+    @skipif_multi_container_tests
+    def test_expected_processes_single_container(
+            self, web_container, worker_container, beat_container):
```

Comment:
Have to pass in those worker/beat containers here so that we've seen the right log line and the processes are in the right state :-/